packages/client3/components/wallet-modal/wallet-modal.tsx

```diff
@@ -21,11 +21,11 @@ export function WalletModal({ isOpen, onClose }: WalletModalProps) {
   }, [isActive, onClose]);
   return (
     <Modal size="xs" title="Select a wallet" isOpen={isOpen} onClose={onClose}>
-      <Paragraph className="mb-8">
+      {/* <Paragraph className="mb-8">
         By connecting your wallet, you agree to our{" "}
         <Link href="/terms">Terms of Service</Link> and{" "}
         <Link href="/privacy">Privacy Policy</Link>
-      </Paragraph>
+      </Paragraph> */}
```

Comment:
Old component used by goldfinch. I commented instead of deleting it for future reference.